### PR TITLE
Rel 22.03

### PR DESCRIPTION
**Pull request by albert17**

### CODE REVIEW

File: docker/inference/dockerfile.ctr [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.2
-ARG TRITON_VERSION=22.01
+ARG TRITON_VERSION=22.02
```
Comment (EvenOldridge) [inline docker/inference/dockerfile.ctr:2]: Do we want to make this a range? >= ?

File: docker/inference/dockerfile.ctr [hunk 2]
```diff
@@ -44,7 +46,7 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 # Install multiple packages
 RUN pip install cupy-cuda115 nvidia-pyindex pybind11 pytest protobuf transformers==4.12 tensorflow-metadata 
 RUN pip install betterproto cachetools graphviz nvtx scipy sklearn
-RUN pip install numba --no-deps
+RUN pip install pandas numba==0.55.1 numpy==1.21.5
```
Comment (EvenOldridge) [inline docker/inference/dockerfile.ctr:49]: Do we need to fix versions here?

File: docker/training/dockerfile.ctr [hunk 3]
```diff
@@ -88,21 +97,16 @@ RUN apt-get update -y && \
...
-    mv ./hadoop-3.3.1 /usr/local/hadoop && cd /usr/local/lib/ && ln -s libhdfs.so.0.0.0 libhdfs.so && \
-    rm /usr/local/hadoop/share/hadoop/common/lib/jackson-databind-2.10.5.1.jar && rm /usr/local/hadoop/share/hadoop/hdfs/lib/jackson-databind-2.10.5.1.jar && \
-    wget https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.13.1/jackson-databind-2.13.1.jar && \
-    cp jackson-databind-2.13.1.jar /usr/local/hadoop/share/hadoop/hdfs/lib/ && \
-    mv jackson-databind-2.13.1.jar /usr/local/hadoop/share/hadoop/common/lib/ && \
-    rm /usr/local/hadoop/share/hadoop/common/lib/log4j-1.2.17.jar && rm /usr/local/hadoop/share/hadoop/hdfs/lib/log4j-1.2.17.jar
+    mv ./hadoop-3.3.1 /usr/local/hadoop && cd /usr/local/lib/ && ln -s libhdfs.so.0.0.0 libhdfs.so ; \
+fi
 
 ENV JAVA_HOME=/usr/java/jdk-16.0.2
```
Comment (EvenOldridge) [inline docker/training/dockerfile.ctr:111]: Do we even need java if we're not installing hdfs?

**Review by EvenOldridge — APPROVED** [review]
Assuming we need to lock versions this looks good to me.  I'm not familiar enough with this to know.

File: docker/inference/dockerfile.ctr (hunk 1 above)
Comment (albert17) [reply docker/inference/dockerfile.ctr:2]: This is a variable, it it not a comparison, it is an assignation

File: docker/inference/dockerfile.ctr (hunk 2 above)
Comment (albert17) [reply docker/inference/dockerfile.ctr:49]: For now we need to change them before installin Merlin Software. We need to update Merlin Software prerequisites

File: docker/training/dockerfile.ctr (hunk 3 above)
Comment (albert17) [reply docker/training/dockerfile.ctr:111]: HugeCTR wanted to keep it as optional. They will be using it for dev

**Pull request merged**